src/openseadragon.js

```diff
@@ -2113,11 +2129,21 @@ function OpenSeadragon( options ){
             try {
                 request.open( "GET", url, true );
 
+                if (responseType) {
+                    request.responseType = responseType;
+                }
+
+                if (headers) {
+                    Object.keys(headers).forEach(function (headerName) {
```

Comment:
I had resolved this but it looks like it got mixed up when I was working from different machines. I'll get that resolved again.